style.css

```diff
@@ -441,87 +476,115 @@ filter: invert(20%);
 }
 
 .text p{
-	margin-top: -5px;
+	margin-top: -7px;
 	font-family: 'Open Sans', sans-serif;
 	font-size: 14px;
 }
 
+/* Image Overlay (Green) */

```

Comment:
A new gradient colour of green, as well as orange and yellow inspired by the gradient overlays used in design for animals and tips.